ios/TPSStripe/TPSStripeManager.m

```diff
@@ -152,14 +305,252 @@ - (NSDictionary *)constantsToExport
                   resolver:(RCTPromiseResolveBlock)resolve
                   rejecter:(RCTPromiseRejectBlock)reject) {
     NSArray <NSString *> *paymentNetworksStrings =
-    options[@"networks"] ?: [StripeModule supportedPaymentNetworksStrings];
+    options[@"networks"] ?: [StripeModule applePaySupportedPaymentNetworksStrings];
 
     NSArray <PKPaymentNetwork> *networks = [self paymentNetworks:paymentNetworksStrings];
     resolve(@([PKPaymentAuthorizationViewController canMakePaymentsUsingNetworks:networks]));
 }
 
+RCT_EXPORT_METHOD(potentiallyAvailableNativePayNetworks:(RCTPromiseResolveBlock)resolve
+                  rejecter:(RCTPromiseRejectBlock)reject) {
+    NSArray <NSString *> *paymentNetworksStrings = [StripeModule applePaySupportedPaymentNetworksStrings];
+
+    NSArray <PKPaymentNetwork> *networks = [self paymentNetworks:paymentNetworksStrings];
+    resolve([PKPaymentAuthorizationViewController canMakePaymentsUsingNetworks:networks] ? paymentNetworksStrings : nil);
+}
+
+
+RCT_EXPORT_METHOD(createPaymentMethod:(NSDictionary<TPSStripeType(createPaymentMethod), id>*)params
+                  resolver:(RCTPromiseResolveBlock)resolve
+                  rejecter:(RCTPromiseRejectBlock)reject) {
+
+    STPPaymentMethodParams * parsed = [self extractCreatePaymentMethodParamsFromDictionary:params];
+    NSParameterAssert(parsed);
+
+    if(!requestIsCompleted) {
+        NSDictionary *error = [errorCodes valueForKey:kErrorKeyBusy];
+        reject(error[kErrorKeyCode], error[kErrorKeyDescription], nil);
+        return;
+    }
+    requestIsCompleted = NO;
+    promiseResolver = resolve;
+    promiseRejector = reject;
+
+    STPAPIClient *api = self.newAPIClient;
+    [api createPaymentMethodWithParams:parsed
+                            completion:^(STPPaymentMethod * __nullable paymentMethod, NSError * __nullable error){
+                                requestIsCompleted = YES;
+
+                                if (error) {
+                                    NSDictionary *jsError = [errorCodes valueForKey:kErrorKeyApi];
+                                    [self rejectPromiseWithCode:jsError[kErrorKeyCode] message:error.localizedDescription];
+                                    return;
+                                }
+                                resolve(paymentMethod);
```

Comment:
Is this supposed to be converted using `resolve([self convertPaymentMethod:paymentMethod]);` ?

Reply:
Nice catch! 
```suggestion
                                resolve([self convertPaymentMethod: paymentMethod]);
```